classes/class-wc-connect-help-view.php

```diff
@@ -310,6 +322,50 @@ public function page() {
 			);
 		}
 
+		/**
+		 * @return array
+		 */
+		protected function get_tax_health_item() {
+			if ( class_exists( 'WC_Taxjar' ) ) {
+				return [
+					'state'              => 'warning',
+					'show_settings_link' => false,
+					'message'            => __( 'TaxJar extension detected. Automatic taxes functionality is disabled', 'woocommerce-services' ),
+				];
+			}
+
+			$store_country = WC()->countries->get_base_country();
+			if ( ! $this->taxjar_integration->is_supported_country( $store_country ) ) {
+				return [
+					'state'              => 'warning',
+					'show_settings_link' => false,
+					'message'            => sprintf( __( 'Your store\'s country (%s) is not supported. Automatic taxes functionality is disabled', 'woocommerce-services' ), $store_country ),
+				];
+			}
+
+			if ( ! wc_tax_enabled() ) {
+				return [
+					'state'              => 'warning',
+					// if the "taxes" functionality is disabled, the "Tax" settings link will redirect to the "general" tab in WC settings.
+					'show_settings_link' => true,
+					'message'            => __( 'The core WooCommerce taxes functionality is disabled. Enable the "Enable tax rates and calculations" setting in the WooCommerce settings page', 'woocommerce-services' ),
```

Comment:
```suggestion
					'message'            => __( 'The core WooCommerce taxes functionality is disabled. Enable the "Enable tax rates and calculations" setting on the WooCommerce settings page', 'woocommerce-services' ),
```

But also I'm not sure about `Enable the "Enable...` bit. How about `Make sure the "Enable..." setting is turned on...`?